### PR TITLE
Fix horizontal scrollbar when float element present

### PR DESCRIPTION
**Pull request by Mathiu**
## Description
`clear: left` was only working for non aligned elements.

## How Has This Been Tested?
Tested on PC Chrome 62 & Firefox 58 (beta5)

## Checklist:
- [x] My code is tested. (partially i guess?)
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: editor/components/block-list/style.scss [hunk 1]
```diff
@@ -400,6 +406,7 @@
 	&:before {
 		content: '';
 		position: absolute;
+		left: 0;
```
Comment (aduth) [inline editor/components/block-list/style.scss]: In my testing, this change alone is sufficient to resolve the issue without the addition of `clear` styles. Can you confirm?

**Comment by aduth** [comment]
Fixes #3612

File: editor/components/block-list/style.scss (hunk 1 above)
Comment (Mathiu) [reply editor/components/block-list/style.scss]: I think you're right, i remember that inserter wasn't wrapping correctly without the `clear` property, but I no longer can reproduce it and it actually wraps the text when there is aligned element and 2 separate paragraph elements after it.

Thanks for looking into it. I have removed unnecessary styles.

**Review by aduth — APPROVED** [review]
In my testing, there is some strange re-layout which occurs when first loading a post with floated content:

relayout

Since I can also reproduce on master, and noting that these changes address the issue of the horizontal scroll, I think it's safe to tackle separately.

**Pull request merged**